### DISCUSSION TITLE
kubernetes-handbook/objects/pod-overview/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#92] [Announcements]
# kubernetes-handbook/objects/pod-overview/

本文将为您讲解 Pod 的基础概念。 理解 Pod Pod 是 kubernetes 中你可以创建和部署的最小也是最简的单位。Pod 代表着集群中运行的进程。 Pod 中封装着应用的容器（有的情况下是好几个容器），存储、独立的网络 IP，管理容器如何运行的策

https://lib.jimmysong.io/kubernetes-handbook/objects/pod-overview/

**Comment by WhistleO** [comment]
包含一个或者多个 Pod 的 Controller 示例：

Deployment
StatefulSet
DaemonSet

三个示例的链接失效了

**Reply by rootsongjc** [reply]
感谢反馈，已修复。
